init-container/index.js

```diff
@@ -106,7 +106,10 @@ async function innerRun() {
         await writeFile(outputFile, serializeToCfgFormatStrict(secrets));
         break;
       case "files":
-        await Promise.all(Object.keys(secrets).map(secretName => writeFile(path.join(program.decryptedPath, secretName), secrets[secretName])))
+        await Promise.all(Object.keys(secrets).map(secretName => writeFile(path.join(program.decryptedPath, secretName), 
```

Comment:
There is a similar code on this [line](https://github.com/Soluto/kamus/blob/b69715467deab9af4f2cf9b505dbc8072fac4ba8/init-container/index.js#L59), maybe worth reusing it?

Reply:
It actually looks like it's doing the opposite of my addition.
I can port my changes to a separate method but I don't think the code can be reused in the current form.

Reply:
The constructor check is a bit magical to me, maybe worth using `typeof`?

Reply:
Good point, I've made the change. Do you also think it's worth moving to a function?

Reply:
Nope, looks better now :)